### PR TITLE
Split & update Android docs to have Table of Contents

### PR DESCRIPTION
**Pull request by todi1856**
* Introduce Table of Contents for Android docs, like other packages do
AndroidDocsReworked

* Juozas to test the docs, open TestProjects\SampleProject1 -> Package manager, select Android Logcat and click button Generate Docs
Docs

* Updated Device Selection, since it was showing old info

### CODE REVIEW

File: com.unity.mobile.android-logcat/Documentation~/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## **Overview**
+
+Android Logcat Package is a utility for displaying log messages coming from Android device in Unity Editor. Read more about [Android Logcat Document](https://developer.android.com/studio/command-line/logcat).
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/index.md:3]: "coming from Android device in Unity Editor"  into "coming from an Android device the Unity Editor"

File: com.unity.mobile.android-logcat/Documentation~/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## **Using Android Logcat**
+
+The toolbar is on the top of the window. Most Android Logcat controls can be found here.  
+![Toolbar](images/android_logcat_toolbar.png)
+
+### Auto Run
+When **Auto Run** is toggled, Android Logcat window will be launched automatically if you do **Build And Run** in **Build Settings** window.
+
+### Stacktrace Resolving
+One benefit of using Android Logcat package is automatic stacktrace resolving. The **addr2line** tool in Android NDK is used to convert the addresses in the crash logs to the file names and line numbers. Below is an example of what's added to log.
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/index.md:40]: "added to log" into "added to the log"

File: com.unity.mobile.android-logcat/Documentation~/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+**_The original log from Android logcat_**
+
+	E CRASH   :      #01  pc 01c65330  /data/app/com.CrashComp.Crash-J2Z_L0XSsSAZPkt9lab2rQ==/lib/arm/libunity.so(DiagnosticsUtils_Bindings::ForceCrash(DiagnosticsUtils_Bindings::ForcedCrashCategory, ScriptingExceptionPtr*)+48)
+
+**_The log shown in Android Logcat window in Unity Editor_**
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/index.md:46]: in the Unity Editor

File: com.unity.mobile.android-logcat/Documentation~/ConnectingToDevice.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+**Connect via USB**  
+The devices connected through USB will be added to the device list automatically when starting Android logcat window. 
+
+**Connect via Wifi**  
+Go to the devices list and click **\<Enter IP>**.
+
+Window to enter an ip address will open.
+
+If you want to manually enter device IP, please follow the instructions in the [Android ADB Document](https://developer.android.com/studio/command-line/adb#wireless) to switch your device to TCP mode first.
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/ConnectingToDevice.md:18]: "manually enter device IP" into "manually enter the device IP"

File: com.unity.mobile.android-logcat/Documentation~/ConnectingToDevice.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+If the device you want to connect through Wifi, is already connected through USB, you can simply click **Connect** .
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/ConnectingToDevice.md:20]: "If the device you want to connect through Wifi, is already connected through USB, you can simply click "

I think we don't need the commas here. Sentence flow seems to be interrupted in the middle of a continuous meaning.

File: com.unity.mobile.android-logcat/Documentation~/Filter.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+#### Filter
+- By Priority  
+  You can right click on the **Priority** column to select or deselect priorities. Priorities are predefined according to the [Android Logcat Document](https://developer.android.com/studio/command-line/logcat#filteringOutput).
+
+- By Tag  
+  Right click on the **Tag** column to select or deselect tags. You can also add or remove a tag by following steps:
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/Filter.md:6]: "remove a tag by following steps" into "remove a tag by following these steps"

File: com.unity.mobile.android-logcat/Documentation~/Installation.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+## **Installing Android Logcat** 
+To install Android Logcat package, follow the instructions in the [Package Manager documentation](https://docs.unity3d.com/Packages/com.unity.package-manager-ui@latest/index.html). 
+
+You can access Android Logcat window in Unity Editor via
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/Installation.md:4]: "the Unity Editor"

File: com.unity.mobile.android-logcat/Documentation~/LogControls.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+![Log Columns](images/log_columns.png)
+
+**Copy**  
+The selected logs can be copied to the clipboard. You can right click on the messages and select **Copy**.
+
+**Save**  
+The selected logs can be saved to file. You can right click on the messages and select **Save Selection**.
+
+**Clear**  
+You can clear all the logs by click the **Clear** button on the toolbar.
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/LogControls.md:13]: "clear all the logs by click the" into "clear all the logs by clicking the"

File: com.unity.mobile.android-logcat/Documentation~/LogControls.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+**Search**  
+You can input text in the text field on the toolbar to search logs by text. You can also toggle **Regex** on to treat contents in the text field as regex expression.
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/LogControls.md:16]: "field as regex expression." into "field as a regex expression."

Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/LogControls.md:16]: Maybe we should let the user know this search is Case sensitive? Also as a disclaimer to state that it's using the search methods of android logcat and that whatever limitations that search has, we have as well? (To save ourselves the trouble of explaining it later)

File: com.unity.mobile.android-logcat/Documentation~/PackageSelection.md [hunk 10]
```diff
@@ -0,0 +1,8 @@
+### Package Selection
+The package selection list contains:
+- Package of the top activity which is currently running on the selected device.
+- Package from the player settings if it's running on the selected device (Doesn't have to be the the top running activity).
+
+You can only select one package at a time, only the log messages coming from the selected package will be shown in the logcat console window. 
+
+Alternatively you can select **No Filter**, messages coming from all package will be shown.
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/PackageSelection.md:8]: "Alternatively you can select **No Filter**, messages coming from all package will be shown." into "Alternatively you can select **No Filter** - messages coming from all packages will be shown."

File: com.unity.mobile.android-logcat/Documentation~/ScreenCapture.md [hunk 11]
```diff
@@ -0,0 +1,3 @@
+### Screen Capture
+You can use **Capture Screen** button on the toolbar to capture a screenshot of the selected device. You can also **Save** the screenshot as file.
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/ScreenCapture.md:2]: "You can use **Capture Screen** button on the toolbar to capture a screenshot of the selected device. You can also **Save** the screenshot as file.  " into "You can use the **Capture Screen** button on the toolbar to capture a screenshot of the selected device. You can also **Save** the screenshot as a file.  "

File: com.unity.mobile.android-logcat/Documentation~/StacktraceUtility.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+Allows you to copy and paste custom log and resolve stacktrace.
+
+Quick Steps:
+- Copy paste a crash log from the application logcat.
+- Set symbol path for specific **Configuration/Scripting Backend/CPU**
+  - For ex., if crash log contains the following line:
+    - **2019-05-17 12:00:58.823 30759-30803/? E/CRASH: Build type 'Release', Scripting Backend 'mono', CPU 'armeabi-v7a'"**
+  - The build will be **Release/mono/armeabi-v7a**
+  - The symbol path will be **Unity_Version/Editor/Data/PlaybackEngines/AndroidPlayer/Variations/mono/Release/Symbols/armeabi-v7a**
+- (Optional)Adjust address resolving regex, the default regex is set to resolve addresses from following line:
```
Comment (JuozasK) [inline com.unity.mobile.android-logcat/Documentation~/StacktraceUtility.md:11]: "(Optional)Adjust address resolving regex, the default regex is set to resolve addresses from following line:" into "(Optional)Adjust the address resolving regex - the default regex is set to resolve addresses from the following line:"

File: com.unity.mobile.android-logcat/Documentation~/LogControls.md (hunk 9 above)
Comment (todi1856) [reply com.unity.mobile.android-logcat/Documentation~/LogControls.md:16]: Makes sense

**Pull request merged**